environment.go

```diff
@@ -882,6 +882,9 @@ func (e *envState) initTracer(ctx context.Context) error {
 	)
 	tp.RegisterSpanProcessor(utility.NewAttributeSpanProcessor())
 	otel.SetTracerProvider(tp)
+	otel.SetErrorHandler(otel.ErrorHandlerFunc(func(err error) {
+		grip.Error(errors.Wrap(err, "encountered otel error"))
```

Comment:
I would set the message as `otel error`.

Reply:
done